tests/openwrt/test_wireless.py

```diff
@@ -732,6 +732,54 @@ def test_parse_wds_bridge(self):
         o = OpenWrt(native=self._wds_bridge_uci)
         self.assertEqual(o.config, self._wds_bridge_netjson)
 
+    _80211r_netjson = {
+        "interfaces": [
+            {
+                "name": "wlan0",
+                "type": "wireless",
+                "wireless": {
+                    "radio": "radio0",
+                    "mode": "access_point",
+                    "ssid": "MyWifiAP",
+                    "ieee80211r": True,
+                    "ft_over_ds": False,
+                    "ft_psk_generate_local": True,
+                    "rsn_preauth": True,
+                    "reassociation_deadline": 1000,
+                },
+            }
+        ]
+    }
+    _80211r_uci = """package network
+
+config interface 'wlan0'
+    option ifname 'wlan0'
+    option proto 'none'
+
+package wireless
+
+config wifi-iface 'wifi_wlan0'
+    option device 'radio0'
+    option ft_over_ds '0'
+    option ft_psk_generate_local '1'
+    option ieee80211r '1'
+    option ifname 'wlan0'
+    option mode 'ap'
+    option network 'wlan0'
+    option reassociation_deadline '1000'
+    option rsn_preauth '1'
+    option ssid 'MyWifiAP'
+"""
+
+    def test_render_access_point_80211r(self):
+        o = OpenWrt(self._80211r_netjson)
+        expected = self._tabs(self._80211r_uci)
+        self.assertEqual(o.render(), expected)
+
+    def test_parse_access_point_80211r(self):
+        o = OpenWrt(native=self._80211r_uci)
+        self.assertEqual(o.config, self._80211r_netjson)
```

Comment:
@nemesisdesign [this test](https://github.com/openwisp/netjsonconfig/runs/5495262212?check_suite_focus=true#step:7:16) I added for 802.11r is failing for some reason. Can you help me identity the issue? 

```
jsonschema.exceptions.ValidationError: OrderedDict([('name', 'wlan0'), ('type', 'wireless'), ('wireless', OrderedDict([('ft_over_ds', '0'), ('ft_psk_generate_local', '1'), ('ieee80211r', '1'), ('mode', 'access_point'), ('reassociation_deadline', '1000'), ('rsn_preauth', '1'), ('ssid', 'MyWifiAP'), ('radio', 'radio0')]))]) is not valid under any of the given schemas
```